dwave/embedding/utils.py

```diff
@@ -18,14 +18,39 @@
 
 import dimod
 import numpy as np
+import math
 
 from six import iteritems
 
 from dwave.embedding.chain_breaks import broken_chains
 
 
-__all__ = 'target_to_source', 'chain_to_quadratic', 'chain_break_frequency', 'adjacency_to_edges'
+__all__ = 'get_chain_strength', 'target_to_source', 'chain_to_quadratic', 'chain_break_frequency', 'adjacency_to_edges'
 
+def get_chain_strength(bqm, prefactor):
+    """Calculates a chain strength scaled to the problem.
+
+    Args: 
+        bqm (:obj:`.BinaryQuadraticModel`):
+            A binary quadratic model.
+
+        prefactor (float):
+            Prefactor used for scaling. For optimal chain strength, the prefactor
+            should fall in the range of [0.5, 2].
```

Comment:
This sounds like a strong assertion. Are we sure about that? Can't I construct a problem for which the optimal chain strength will be 3?

Reply:
I think that for non-pathological problems (that don't require extreme high precision) this is generally true. A good way to put it is that when optimizing it, a good starting point is to perform a grid search over that range. I tried with 3 kinds of problem (binomial Jij, RAN1 and other user applications) and it was always falling in that range, actually larger than 1, never bigger than 2.

Reply:
Then I think we should qualify the claim with "for non-pathological problems" (or similar).